bin/backup

```diff
@@ -172,16 +172,16 @@ while getopts 'hc:' ARG; do
     case ${ARG} in
         'h')
             cat <<-"EOF"
-			usage: backup [OPTIONS] [MODULE...]
```

Comment:
The tabs are actually syntactic here: https://www.gnu.org/software/bash/manual/bashref.html#Here-Documents

After this change, the backup script no longer runs:
> ./bin/backup: line 232: warning: here-document at line 174 delimited by end-of-file (wanted `EOF')
./bin/backup: line 233: syntax error: unexpected end of file

I'm using bash 4.3.30, fwiw.